docs/how-to/default-profile.md

```diff
@@ -85,7 +85,7 @@ rm /tmp/ipfs-ds-convert.tar.gz
 
 ### Linux
 
-We will download the tarball for Linux, extract the contents, and move the binary file to our path
+You download the tarball for Linux, extract the contents, and move the binary file to your path:
```

Comment:
```suggestion
Download the tarball for Linux, extract the contents, and move the binary file to your path:
```

Let's turn this into an instruction rather than a cometary.